Build failure: virt-manager-qt

### Steps To Reproduce

Steps to reproduce the behavior:
1. build *virt-manager-qt* on 24.05 (or, I assume, unstable)

### Build log

Full log: https://hydra.nixos.org/build/260898426/nixlog/4

```
-- Checking for module 'qtermwidget5'
--   No package 'qtermwidget5' found
CMake Error at /nix/store/pb69k0zwhxg6kf9ifz13x1wjs6a9vmf0-cmake-3.29.2/share/cmake-3.29/Modules/FindPkgConfig.cmake:634 (message):
  The following required packages were not found:

   - qtermwidget5
```

### Additional context

Bisecting the build yields 9ee17ce743b4ba53878fcad5639f6b3ee057177a from #313122  as the first bad commit. It updates `lxqt.qtermwidget` from `1.4.0` to `2.0.0`.

### Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@peterhoeg @romildo 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/virtualization/virt-manager/qt.nix
+++ b/pkgs/applications/virtualization/virt-manager/qt.nix
@@ -1,4 +1,4 @@
-{ mkDerivation, lib, fetchFromGitHub, fetchpatch, cmake, pkg-config
+{ mkDerivation, lib, fetchFromGitHub, cmake, pkg-config
 , qtbase, qtmultimedia, qtsvg, qttools, krdc
 , libvncserver, libvirt, pcre, pixman, qtermwidget, spice-gtk, spice-protocol
 , libselinux, libsepol, util-linux
@@ -6,26 +6,18 @@
 
 mkDerivation rec {
   pname = "virt-manager-qt";
-  version = "0.72.97";
+  version = "0.72.99";
 
   src = fetchFromGitHub {
     owner  = "F1ash";
     repo   = "qt-virt-manager";
     rev    = version;
-    sha256 = "0b2bx7ah35glcsiv186sc9cqdrkhg1vs9jz036k9byk61np0cb1i";
+    hash   = "sha256-1aXlGlK+YPOe2X51xycWvSu8YC9uCywyL6ItiScFA04=";
   };
 
   cmakeFlags = [
     "-DBUILD_QT_VERSION=5"
     "-DQTERMWIDGET_INCLUDE_DIRS=${qtermwidget}/include/qtermwidget5"
-  ];
-
-  patches = [
-    (fetchpatch {
-      # drop with next update
-      url = "https://github.com/F1ash/qt-virt-manager/commit/0d338b037ef58c376d468c1cd4521a34ea181edd.patch";
-      sha256 = "1wjqyc5wsnxfwwjzgqjr9hcqhd867amwhjd712qyvpvz8x7p2s24";
-    })
   ];
 
   buildInputs = [

--- a/pkgs/desktops/lxqt/default.nix
+++ b/pkgs/desktops/lxqt/default.nix
@@ -75,6 +75,11 @@ let
       libfm-qt = libfm-qt_1_4;
       inherit (pkgs.libsForQt5) qtbase qtsvg qttools libdbusmenu;
     };
+    qtermwidget_1_4 = callPackage ./qtermwidget {
+      version = "1.4.0";
+      lxqt-build-tools = lxqt-build-tools_0_13;
+      inherit (pkgs.libsForQt5) qtbase qttools;
+    };
 
     preRequisitePackages = [
       kdePackages.kwindowsystem # provides some QT plugins needed by lxqt-panel

--- a/pkgs/desktops/lxqt/qtermwidget/default.nix
+++ b/pkgs/desktops/lxqt/qtermwidget/default.nix
@@ -7,17 +7,21 @@
 , lxqt-build-tools
 , wrapQtAppsHook
 , gitUpdater
+, version ? "2.0.0"
 }:
 
 stdenv.mkDerivation rec {
   pname = "qtermwidget";
-  version = "2.0.0";
+  inherit version;
 
   src = fetchFromGitHub {
     owner = "lxqt";
     repo = pname;
     rev = version;
-    hash = "sha256-kZS6D/wSJFRt/+Afq0zCCmNnJPpFT+1hd4zVPc+rJsE=";
+    hash = {
+      "1.4.0" = "sha256-wYUOqAiBjnupX1ITbFMw7sAk42V37yDz9SrjVhE4FgU=";
+      "2.0.0" = "sha256-kZS6D/wSJFRt/+Afq0zCCmNnJPpFT+1hd4zVPc+rJsE=";
+    }."${version}";
   };
 
   nativeBuildInputs = [

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -35176,7 +35176,7 @@ with pkgs;
   };
 
   virt-manager-qt = libsForQt5.callPackage ../applications/virtualization/virt-manager/qt.nix {
-    qtermwidget = lxqt.qtermwidget;
+    qtermwidget = lxqt.qtermwidget_1_4;
   };
 
   virtscreen = callPackage ../tools/admin/virtscreen { };

